docs/docs/getting-started/installation/aws.md

```diff
@@ -128,15 +128,68 @@ Error creating vpc:  operation error EC2: CreateVpc, https response error Status
 When encountering this, you can delete some of the resources just as described
 on [here](/getting-started/installation/aws#uninstall-airy-core)
 
+## Enable HTTPS
+
+This section explains all the steps needed to configure HTTPS on your `Airy Core` instance.
+
+Create a self-signing certificate (skip step if you already have your existing https certificate).
+The certificates will be created in your `~/.airy/certs` directory, using the OpenSSL utility.
+
+```sh
+mkdir -p ~/.airy/certs/
+cd ~/.airy/certs/
+openssl genrsa 2048 > private.key
+openssl req -new -x509 -nodes -sha1 -days 3650 -extensions v3_ca -key private.key > public.crt
+-----
+Country Name (2 letter code) [DE]:DE
+State or Province Name (full name) [Berlin]:Berlin
+Locality Name (eg, city) []:Berlin
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:Airy GmbH
+Organizational Unit Name (eg, section) []:Development
+Common Name (e.g. server FQDN or YOUR name) []:*.elb.amazonaws.com
+Email Address []:sre@airy.co
+```
+
+Note that when generating the public certificate `public.crt` you must specify a common namd of fqdn, otherwise the certificate will not be used by the AWS LoadBalancer.
+
+Next you should upload your certificates to the AWS certificates manager (ACM).
+
+```sh
+aws acm import-certificate --certificate fileb://public.crt --private-key fileb://private.key --region us-east-1
+```
+
+In case you have your own certificates and have one more file with the certificate chain, you can also specify that file as anadditional `--certificate` value.
```

Comment:
```suggestion
In case you have your own certificates and have one more file with the certificate chain, you can also specify that file as additional `--certificate` value.
```

It's a little hard to understand what's that "that file" in the second sentence so maybe we should reword a little